### PR TITLE
CUMULUS-4086: Add test case/fixture/validate dashboard handles duplicate producerGranuleId correctly in all displays.

### PR DESCRIPTION
**Pull request by Nnaga1**
**Summary:** Summary of changes

Added granule fixtures and cypress test assertions to test if duplicate producerGranuleId's persist in granules' tables.

## PR Checklist

- [X] Update CHANGELOG
- [X] Unit tests
- [X] Adhoc testing
- [X] Integration tests

### CODE REVIEW

File: cypress/fixtures/seeds/granulesFixture.json [hunk 1]
```diff
@@ -140,6 +141,58 @@
...
+        },
+        {
+          "bucket": "cumulus-test-sandbox-protected-2",
+          "key": "MOD09GQ___006/MOD/MOD09GQ.A1657416.CbyoRi.006.9697917818587.cmr.xml",
+          "size" : 124521,
+          "fileName": "MOD09GQ.A1657416.CbyoRi.006.9697917818587.cmr.xml"
+        }
+      ],
+      "collectionId": "MOD09GQ___006",
+      "producerGranuleId": "MOD09GQ.A1657416.CbyoRi.006.9697917818587",
```
Comment (jennyhliu) [inline cypress/fixtures/seeds/granulesFixture.json:191]: Both granules with `"producerGranuleId": "MOD09GQ.A1657416.CbyoRi.006.9697917818587"` from `"collectionId": "MOD09GQ___006"`, please add one more granule from a different collection.

File: cypress/integration/granules_spec.js [hunk 2]
```diff
@@ -192,8 +192,15 @@ describe('Dashboard Granules Page', () => {
           }
         });
 
+      // checks that duplicate producerIds can exist in the table
+      cy.get('@granulesListFixture').its('results').then((results) => {
```
Comment (jennyhliu) [inline cypress/integration/granules_spec.js:196]: This doesn't test the table, it just checks the fixture.

**Review by jennyhliu — APPROVED** [review]
Nice work!
